pkg/os/os.go

```diff
@@ -90,7 +91,11 @@ func (RealOS) Mount(source string, target string, fstype string, flags uintptr,
 	return unix.Mount(source, target, fstype, flags, data)
 }
 
-// Unmount will call unix.Unmount to unmount the file.
+// Unmount will call unix.Unmount to unmount the file. The function doesn't
+// return error if target is not mounted.
 func (RealOS) Unmount(target string, flags int) error {
+	if mounted, err := mount.Mounted(target); err != nil || !mounted {
```

Comment:
We put everything in `/var/lib` now. :p May need to consider the directory structure more later.
Will add a TODO here to take care of this in the future.